jscomp/test/ImportAttributes.res

```diff
@@ -0,0 +1,6 @@
+@@config({flags: ["-bs-package-output", "es6:jscomp/test:.mjs"]})
+
+@module({from: "./myJson.json", type_: "json"})
```

Comment:
I prefer a syntax that explicitly provides the "with" key rather than having the special keys (that is the whole point of the spec over the import asertion) .

This is a common even in regular js syntax.

```js
// import attrs in dynamic imports
import("module-id", { with: { ... } })

// import attra in workers
new Worker("id", { with: { ... } })

// Notice that the "type" in the top-level and the "type" in the import attributes (with) have different semantic
new Worker("id", { type: "module", with: { type: "something-special" } })
```

Reply:
Can you write an example of how you'd want it to look?

Reply:
```
@module({from: "./myJson.json", with: { type_: "json"}})
```

Reply:
Right. Curious, can more things appear in that with list than `type`?

Reply:
Ok @cometkim look again.

Reply:
> Right. Curious, can more things appear in that with list than `type`? 

Yes, import attributes are freely extensible and their behavior is determined by the host (mostly bundlers).

Reply:
@cometkim is the input/output fine now?

Reply:
yep, it will be better if we have other random cases than `type_`